src/reanimated2/NativeReanimated/NativeReanimated.ts

```diff
@@ -1,31 +1,34 @@
+import { NativeModules } from 'react-native';
 import { SharedValue } from '../commonTypes';
 import { Descriptor } from '../hook/commonTypes';
 
-const InnerNativeModule = global.__reanimatedModuleProxy;
-
 export class NativeReanimated {
   native: boolean;
-  useOnlyV1: boolean;
```

Comment:
why is this no longer needed?

Reply:
This property was true only if Reanimated's turbomodule wasn't loaded.`useOnlyV1` was used:
1. To guard `NativeReanimatedModule.installCoreFunctions()` call, which setups Rea2 specific functions
2. inside `isConfigured()`, checking if Rea2 was correctly installed. `isConfigured()` is used only to print errors and install global bindings (`_setGlobalConsole`, `global.performance`).

I'll check if the web platform still works though.